src/main/java/org/team1540/robot2018/Robot.java

```diff
@@ -95,44 +87,7 @@ public void robotInit() {
     zeroElevator.setRunWhenDisabled(true);
     SmartDashboard.putData(zeroElevator);
 
-    // configure controls
-    OI.autoIntakeButton.whenPressed(new IntakeSequence());
-    // OI.autoIntakeButton.whileHeld(new SimpleCommand("Intake Arm Open", () -> intakeArms.set
-    //     (Tuning.intakeArmSpeed), intakeArms));
-    OI.autoIntakeButton.whileHeld(new JoystickArms());
-
-    OI.autoEjectButton.whenPressed(new EjectCube());
-    OI.stopIntakeButton.whenPressed(new SimpleCommand("Stop intake", intake::stop, intake,
-        intakeArms));
-
-    OI.elevatorExchangeButton.whenPressed(new MoveElevatorToPosition(Tuning
-        .elevatorExchangePosition));
-
-    OI.elevatorSwitchButton.whenPressed(new MoveElevatorToPosition(Tuning
-        .elevatorFrontSwitchPosition));
-    // OI.elevatorRaiseButton.whenPressed(new MoveElevatorToPosition(Tuning.elevatorScalePosition));
-    OI.elevatorFrontScaleButton.whenPressed(new FrontScale());
-    OI.elevatorLowerButton.whenPressed(new GroundPosition());
-
-    OI.wristFwdButton.whenPressed(new CalibrateWrist());
-    OI.wrist45DegButton.whenPressed(new MoveWristToPosition(Tuning.wrist45FwdPosition));
-    OI.wristBackButton.whenPressed(new MoveWristToPosition(Tuning.wristBackPosition));
-
-    OI.enableElevatorAxisControlButton.whileHeld(new JoystickElevator());
-    OI.enableWristAxisControlButton.whileHeld(new JoystickWrist());
-
-    OI.holdElevatorWristButton.whenPressed(new HoldElevatorWrist());
-
-
-    OI.winchInSlowButton.whileHeld(new SimpleCommand("Winch In Low", () -> winch.set(Tuning
-        .winchInLowVel), winch));
-
-    OI.winchInFastButton.whileHeld(new SimpleCommand("Winch In High", () -> winch.set(Tuning
-        .winchInHighVel), winch));
-
-    // OI.climbSequenceButton.whenPressed(new ClimbSequence());
-
-    // configure camera crosshairs
+    // TODO: Move camera crosshairs into separate (command?)
```

Comment:
Not really–doesn't require any subsystems since it doesn't actually exist, and will always be running, so no big deal

Reply:
Separate class to start the thread?

Reply:
No good reason, just unnecessary lines of code